Chore/update oss api (#3341)
influxdata/docs-v2: content/enterprise_influxdb/v1.8/concepts/clustering.md

    parent: Concepts
---

This document describes in detail how clustering works in InfluxDB Enterprise.

> It starts with a high level description of the different components of a cluster
> and then outlines implementation details.

Reviewer: ```suggestion
Includes a high-level description of cluster components and outlines implementation details.
```
